pkg/cluster/node.go

```diff
@@ -83,9 +96,9 @@ func createNode(name, image, clusterLabel string) (handle *nodeHandle, err error
 		handle = &nodeHandle{name}
 	}
 	if err != nil {
-		return handle, errors.Wrap(err, "docker run error")
+		return handle, 0, errors.Wrap(err, "docker run error")
 	}
-	return handle, nil
+	return handle, port, nil
```

Comment:
Add port to the nodeHandle struct?

Reply:
the port is totally orthogonal to the node handle.